### ISSUE TITLE
governance: categorization of items

### ISSUE THREAD

**Issue by mochet** [Joystream/community-repo#713]
This issue is a tracking issue, I am working on some areas of governance and exploring topics (which is not on GitHub or any public space yet). I attempt for these to be neutral but they are obviously my own ideas written based on my own interpretations. Nothing is finalized at all--these are just topics and anything that may be introduced would follow the ordinary route that previous governance proposals have taken, including a discussion phase.

Summary:
- There is a `labels` feature that I haven't been able to access yet, which allows users to categorize proposals. Depending on how this feature works it may change what is required on this item.
- How exactly governance will be structured is a complete mystery. Depending on how that develops over time would impact this.
- Categorization should occur after the issuance of any item
- Every proposal or item should be categorized to help with organization
- `Council Secretary` or a similar role would maybe be responsible for doing this
- Categorization might include scope (WGs, Leads, Workers, Payments), type (policy, item, rule, constitution), budget categorization (WGs, discretionary spending, council spending, specific projects) and many others
- Each item should also include date, author and other metadata

**Comment by mochet** [comment]
Relevant: https://github.com/Joystream/community-repo/issues/509

**Comment by traumschule** [comment]
> There is a labels feature that I haven't been able to access yet, which allows users to categorize proposals

I don't think this is correct. According to current release plan (https://github.com/Joystream/joystream/issues/2949) only threads will support  labels from Olympia onwards: https://github.com/Joystream/joystream/issues/2422
However proposals can have threads  https://github.com/Joystream/joystream/pull/2547 which could be what you are looking for?
